game/src/edit/roads.rs

```diff
@@ -508,8 +508,34 @@ fn can_reverse(_: LaneType) -> bool {
     lt == LaneType::Driving || lt == LaneType::Biking || lt == LaneType::Bus
 }*/
 
+fn default_outside_lane_placement(road: &mut EditRoad, dir: Direction) -> usize {
+    let idx = if road.lanes_ltr.first().unwrap().dir == dir {
+        if road.lanes_ltr.first().unwrap().lt.is_walkable() {
+            1
+        } else {
+            0
+        }
+    } else {
+        if road.lanes_ltr.last().unwrap().lt.is_walkable() {
+            road.lanes_ltr.len() - 1
+        } else {
+            road.lanes_ltr.len()
+        }
+    };
+    idx
```

Comment:
Style: You could just make the entire function be an expression, aka, take away the `let idx = ` part